controllers/platformnetwork_controller.go

```diff
@@ -12,8 +12,10 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/starlingx/inventory/v1/addresspools"
+	gcHosts "github.com/gophercloud/gophercloud/starlingx/inventory/v1/hosts"
 	"github.com/gophercloud/gophercloud/starlingx/inventory/v1/networks"
 	"github.com/gophercloud/gophercloud/starlingx/inventory/v1/oamNetworks"
+	gcSystem "github.com/gophercloud/gophercloud/starlingx/inventory/v1/system"
```

Comment:
You can use system, hosts instead of gcHosts and gcSystem following the pattern

Reply:
Updated accordingly.